### ISSUE TITLE
Escaping of reserved words in schema name

### ISSUE THREAD

**Issue by ishenkoyv** [pomm-project/Cli#32]
If we create schema with name "user" cli generator should create relations with escaped schema name as it's reserved word. E.g.

AutoStructure/Table.php should contains ->setRelation('"user".table') instead of  ->setRelation('user.table')

**Comment by stood** [comment]
I am watching this bug and what's the best solutions :
- Always escape the schema name
- escape the schema name if the word is reserved

**Comment by tlode** [comment]
Hi @stood, I vote for he first solution. The generator should always escape. But why not the relation name too? This way, there wouldn't be the need to maintain a reserved keyword list. Uppercase relation names would safely work too (perhaps they already do, I don't know).

**Comment by stood** [comment]
Hi @tlode I'm agree with you

**Comment by stood** [comment]
And https://github.com/pomm-project/ModelManager/commit/c2b531c071b0d60e5cecda9701506e7de791bbc0 :)

**Comment by chanmix51** [comment]
yes, this is an issue in itself, but re-factoring this means changing Cli, PommBundle and PommModule so we will wait for the next release branch to do that.

**Comment by chanmix51** [comment]
The commits have been rollbacked. They induce hard to track problems in projections and embedded entities. This problem will be addressed in the 2.1 release as it require a bit of refactoring.
